src/utils/update/review_trusts.ts

```diff
@@ -8,15 +8,15 @@ export const hasTrustsToReview = (appData: ApplicationData) =>
 export const getTrustInReview = (appData: ApplicationData) =>
   (appData.update?.review_trusts ?? []).find(trust => !!trust.review_status?.in_review);
 
-export const putTrustInReview = (appData: ApplicationData) => {
+export const setupNextTrustForReview = (appData: ApplicationData) => {
   const trustToReview = (appData.update?.review_trusts ?? [])[0];
 
   if (!trustToReview) {
     return false;
   }
 
   trustToReview.review_status = {
-    in_review: true,
```

Comment:
the trust should be in_review when first selected I think.

Reply:
I believe it is set to false to begin with as not to skip to the BO page. Then setting it to true when landing on the review the trust page. Think this was added to counteract a bug that skipped to the BO page when going back and fourth with backlinks.

Reply:
ok but if the review is done and the user returns here will it not force another review? Unless it remains true - in which case the var name is not ideal but can live with that.

Reply:
@AndreKramer7654 You're right the name isn't ideal; it was a scenario I didn't account for when defining the review status.

You can see in the commit history it was actually I who made this change. I was going to update the name separately in a follow up set of PRs. I just didn't want this to potentially hold up Keelan/Christy's PR going in.

Reply:
FYI the PR to address this is here @AndreKramer7654 https://github.com/companieshouse/overseas-entities-web/pull/1133

Reply:
great thanks. Could we add saveTrustInApp and getTrustByIdFromApp to the list of possible renames?getTrustByIdFromAppData/saveTrustInAppData would better or use Session instead of App.